src/CoreShop/Bundle/TierPricingBundle/Resources/config/pimcore/admin.yml

```diff
@@ -0,0 +1,19 @@
+core_shop_tier_pricing:
+    pimcore_admin:
+        js:
+            tier_pricing_clipboard: '/bundles/coreshoptierpricing/pimcore/js/clipboardManager.js'
+            tier_pricing_core_extension_data_specific_tier_price: '/bundles/coreshoptierpricing/pimcore/js/coreExtension/data/coreShopProductSpecificTierPriceRules.js'
+            tier_pricing_core_extension_tag_specific_tier_price: '/bundles/coreshoptierpricing/pimcore/js/coreExtension/tags/coreShopProductSpecificTierPriceRules.js'
+            tier_pricing_product_specific_tier_price_object_item: '/bundles/coreshoptierpricing/pimcore/js/specifictierprice/object/item.js'
+            tier_pricing_product_specific_tier_price_rule_ranges: '/bundles/coreshoptierpricing/pimcore/js/specifictierprice/ranges.js'
+            tier_pricing_product_specific_tier_price_rule_condition: '/bundles/coreshoptierpricing/pimcore/js/specifictierprice//condition.js'
+            tier_pricing_product_specific_tier_price_condition_nested: '/bundles/coreshoptierpricing/pimcore/js/specifictierprice/conditions/nested.js'
+            tier_pricing_product_specific_tier_price_condition_timespan: '/bundles/coreshoptierpricing/pimcore/js/specifictierprice/conditions/timespan.js'
+            # tier_pricing_product_specific_tier_price_condition_countries: '/bundles/coreshoptierpricing/pimcore/js/specifictierprice/conditions/countries.js'
```

Comment:
needed?

Reply:
yes, currently not working since all condition classes (extjs) are inheriting from corebundle => which is not available at this very moment.